kai_mcp_solution_server/tests/test_multiple_integration.py

```
@@ -353,3 +359,183 @@
+                        }
+                    ],
+                    "reasoning": None,
+                    "used_hint_ids": None,
+                },
+            )
+            SOLUTION_FOR_INCIDENT_A_ID = int(
+                create_solution_for_incident_a.model_dump()["content"][0]["text"]
+            )
+
+    async def test_multiple_users(self) -> None:
+        multiple_user_mcp_args = MCPClientArgs(
```

Ruff (0.12.2)

Local variable `multiple_user_mcp_args` is assigned to but never used

Remove assignment to unused variable `multiple_user_mcp_args`

(F841)